feat(apple): Add more information about source context for cocoa SDK (#7286)
getsentry/sentry-docs: src/platforms/common/data-management/debug-files/source-context.mdx

</PlatformSection>

<PlatformSection supported={["apple"]}>

> To get source context for your Apple project, follow the instructions in the [Uploading Debug Symbols](/platforms/apple/dsym/#pploading-source-context) guide.

Reviewer: ```suggestion
To get source context for your Apple project, follow the instructions in the [Uploading Debug Symbols](/platforms/apple/sourcecontext) guide.
```

Since source context has its own page now, we can link it there.

</PlatformSection>

<PlatformSection supported={["apple"]}>

> <Note>
> 
> TO get source context for you Apple project, follow the instructions in the [Uploading Debug Symbols](/platforms/apple/dsym/#pploading-source-context) guide.
> 
> </Note>

Reviewer 2: ```suggestion
To get source context for your Apple project, follow the instructions in the [Uploading Debug Symbols](/platforms/apple/dsym/#pploading-source-context) guide.
```
